docs/release-notes/2.32.0.md

```diff
@@ -0,0 +1,112 @@
+
+Highlights of the new Home Dashboard include:
+
+- Optimized performance to handle a large number of nodes concurrently
+- Scalable to accomodate varying loads at all times
+- Effortless anomaly detection to check the health of your nodes from the Home dashboard literally at the click of a button.
+- Color-coded states on panels for a quick visual representation of the problem areas
+- Easy-to-use dashboard to reduce cognitive load
+ 
+
+### Enhanced main menu
+
+- In PMM 2.32.0, you can easily access all dashboards for monitored services due to the contemporized navigation structure. All the dashboards in the services folder that were not accessible from the main menu are now accessible from the main menu via **Other Dashboards**. Also, your custom dashboards can be accessed easily from the main menu via **Other Dashboards**
+
+    For example, to access the dashboards for MYSQL, go to *MYSQL >> Other dashboards*.
+
+- Only monitored services are now included in the main menu, enhancing the user experience and minimizing cognitive load to maximize usability.
+
+    For example, you will only see MongoDB dashboards in the main menu if you monitor the MongoDB database rather than all the other dashboards.
+
+## New Features
+
+- [PMM-8792](https://jira.percona.com/browse/PMM-8792), [PMM-9055](https://jira.percona.com/browse/PMM-9055), [PMM-9056](https://jira.percona.com/browse/PMM-9056): PMM 2.32.0 introduces the parameter `--max-query-length` in the `pmm-admin` command for MYSQL, PostreSQL, and MongoDB databases. Now, queries longer than 2048 characters are not truncated (if parameter is set `-1`, which means no limit, or value is bigger than 2048), and the Explain and Examples tabs on the QAN dashboard do not throw an error [PMM-1899](https://jira.percona.com/browse/PMM-1899). These tabs provide contextual information pertaining to the query, thus enhancing user experience.
+
+- [PMM-9545](https://jira.percona.com/browse/PMM-9545): For effective monitoring we have added an additional metric for k8s cluster added in PMM.
+ 
+## Improvements
+
+- [PMM-10156](https://jira.percona.com/browse/PMM-10156): Starting with PMM 2.32.0, we have an enhanced PMM dashboard that gives a bird-view of the services, infrastructure, and critical issues (if any). Thus, you can effortlessly diagnose the problem from the home dashboard with just one click.
+
+  
+- [PMM-9946](https://jira.percona.com/browse/PMM-9946): To enhance user experience, we have simplified the deployment of PMM inside the K8s cluster using Manifest or Helm.
+
+
+- [PMM-10393](https://jira.percona.com/browse/PMM-10393): Panel for total time taken for scraping metrics by Postgres exporter is improved. Instead of showing coarse-grained measurements of the total time taken by the exporter, it now shows the total time taken per collector.
+
+- [PMM-10555](https://jira.percona.com/browse/PMM-10555): Add meta-metric support
```

Comment:
@ShashankSinha252 @denisok Could you please provide description for this ticket.

Reply:
Runtime for collectors in MongoDB exporter is pushed via Telemetry to PMM portal service. This information is used to detect unusually large scrape time for collector and improve collector performance in future releases.